[2023-12-15 23:11:30,518: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-15 23:11:30,526: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-15 23:11:30,539: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-15 23:11:30,539: INFO: common: created directory at: artifacts]
[2023-12-15 23:16:37,214: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-15 23:16:37,214: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-15 23:16:37,219: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-15 23:16:37,219: INFO: common: created directory at: artifacts]
[2023-12-15 23:18:26,510: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-15 23:18:26,526: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-15 23:18:26,526: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-15 23:18:26,532: INFO: common: created directory at: artifacts]
[2023-12-15 23:18:26,532: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-15 23:18:28,256: INFO: 2353853782: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: C5A0:3EDF:58A480:6C0755:657C916A
Accept-Ranges: bytes
Date: Fri, 15 Dec 2023 17:48:27 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10238-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1702662507.644277,VS0,VE787
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 599b9b0ac4f7e148352aa43892a0cb8e77ce60d3
Expires: Fri, 15 Dec 2023 17:53:27 GMT
Source-Age: 0

]
[2023-12-30 00:47:00,512: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-30 00:47:00,513: ERROR: main: name 'DataIngestionTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 6, in <module>
    data_ingestion = DataIngestionTrainingPipeline()
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'DataIngestionTrainingPipeline' is not defined
[2023-12-30 00:47:36,027: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-30 00:47:36,036: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-30 00:47:36,037: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-30 00:47:36,042: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-30 00:47:36,043: INFO: common: created directory at: artifacts]
[2023-12-30 00:47:36,044: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-30 00:47:38,810: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7E02:04A0:76064A:90A0B0:658F1B51
Accept-Ranges: bytes
Date: Fri, 29 Dec 2023 19:17:38 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10243-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1703877457.941676,VS0,VE1546
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: f0217e8d52748f76516ecd013f439405caa16827
Expires: Fri, 29 Dec 2023 19:22:38 GMT
Source-Age: 0

]
[2023-12-30 00:47:38,827: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-12-30 00:49:10,392: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-30 00:49:10,394: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-30 00:49:10,395: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-30 00:49:10,396: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-30 00:49:10,399: INFO: common: created directory at: artifacts]
[2023-12-30 00:49:10,399: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-30 00:49:11,494: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7E02:04A0:76064A:90A0B0:658F1B51
Accept-Ranges: bytes
Date: Fri, 29 Dec 2023 19:19:11 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10251-MAA
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1703877551.198617,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: c875d4a20a235a6698512f5890bced71529300ec
Expires: Fri, 29 Dec 2023 19:24:11 GMT
Source-Age: 93

]
[2023-12-30 00:49:11,506: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-12-30 03:10:06,289: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-30 03:10:06,290: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-30 03:10:06,291: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-30 03:10:06,296: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-30 03:10:06,297: INFO: common: created directory at: artifacts]
[2023-12-30 03:10:06,297: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-30 03:10:08,471: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 998C:5B9B:6E2FC4:8BC3A5:658F3CB6
Accept-Ranges: bytes
Date: Fri, 29 Dec 2023 21:40:08 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10240-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1703886007.197424,VS0,VE957
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: e842fd95adcc355dc4105b408f88a3eaff06f735
Expires: Fri, 29 Dec 2023 21:45:08 GMT
Source-Age: 1

]
[2023-12-30 03:10:08,485: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-12-31 04:00:19,781: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 04:00:19,789: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 04:00:19,801: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 04:00:19,804: INFO: common: created directory at: artifacts]
[2023-12-31 04:00:19,805: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 13:55:24,065: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 13:55:24,073: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 13:55:24,081: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 13:55:24,081: INFO: common: created directory at: artifacts]
[2023-12-31 13:55:24,081: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 14:00:09,611: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:00:09,615: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:00:09,623: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 14:00:09,625: INFO: common: created directory at: artifacts]
[2023-12-31 14:00:09,628: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 14:01:36,035: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:01:36,037: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:01:36,052: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 14:01:36,057: INFO: common: created directory at: artifacts]
[2023-12-31 14:01:36,057: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 14:03:35,419: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:03:35,420: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:03:35,428: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 14:03:35,430: INFO: common: created directory at: artifacts]
[2023-12-31 14:03:35,430: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 14:11:31,460: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:11:31,463: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:11:31,467: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 14:11:31,470: INFO: common: created directory at: artifacts]
[2023-12-31 14:11:31,472: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 14:16:23,023: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:16:23,023: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 14:16:23,031: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 14:16:23,036: INFO: common: created directory at: artifacts]
[2023-12-31 14:16:23,036: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 17:52:10,624: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 17:52:10,627: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 17:52:10,639: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 17:52:10,641: INFO: common: created directory at: artifacts]
[2023-12-31 17:52:10,643: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 18:01:31,193: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 18:01:31,196: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 18:01:31,201: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 18:01:31,202: INFO: common: created directory at: artifacts]
[2023-12-31 18:01:31,204: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 19:33:01,801: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 19:33:01,803: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 19:33:01,807: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 19:33:01,807: INFO: common: created directory at: artifacts]
[2023-12-31 19:33:01,812: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 19:43:03,953: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 19:43:03,956: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 19:43:03,965: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 19:43:03,968: INFO: common: created directory at: artifacts]
[2023-12-31 19:43:03,970: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-31 19:52:31,426: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 19:52:31,429: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 19:52:31,430: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 19:52:31,435: INFO: common: created directory at: artifacts]
[2023-12-31 19:52:31,437: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:21:40,841: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:21:40,844: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:21:40,847: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:21:40,849: INFO: common: created directory at: artifacts]
[2023-12-31 20:21:40,850: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:22:51,288: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:22:51,291: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:22:51,295: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:22:51,297: INFO: common: created directory at: artifacts]
[2023-12-31 20:22:51,299: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:25:31,635: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:25:31,639: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:25:31,643: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:25:31,645: INFO: common: created directory at: artifacts]
[2023-12-31 20:25:31,647: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:26:04,583: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:26:04,585: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:26:04,588: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:26:04,588: INFO: common: created directory at: artifacts]
[2023-12-31 20:26:04,594: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:26:25,506: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:26:25,510: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:26:25,513: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:26:25,516: INFO: common: created directory at: artifacts]
[2023-12-31 20:26:25,518: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:27:14,111: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:27:14,115: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:27:14,119: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:27:14,122: INFO: common: created directory at: artifacts]
[2023-12-31 20:27:14,123: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:49:24,568: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:49:24,571: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:49:24,575: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:49:24,576: INFO: common: created directory at: artifacts]
[2023-12-31 20:49:24,578: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 20:53:34,837: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:53:34,840: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:53:34,843: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:53:34,846: INFO: common: created directory at: artifacts]
[2023-12-31 20:53:56,525: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:53:56,526: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 20:53:56,534: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 20:53:56,536: INFO: common: created directory at: artifacts]
[2023-12-31 21:06:42,328: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 21:06:42,331: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 21:06:42,338: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 21:06:42,340: INFO: common: created directory at: artifacts]
[2023-12-31 21:08:03,081: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 21:08:03,083: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 21:08:03,083: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 21:08:03,091: INFO: common: created directory at: artifacts]
[2023-12-31 21:12:24,631: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 21:12:24,634: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 21:12:24,638: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 21:12:24,640: INFO: common: created directory at: artifacts]
[2023-12-31 21:12:24,641: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 22:15:37,500: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 22:15:37,505: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 22:15:37,516: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 22:15:37,521: INFO: common: created directory at: artifacts]
[2023-12-31 22:15:37,521: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 22:41:52,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 22:41:52,462: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 22:41:52,470: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 22:41:52,470: INFO: common: created directory at: artifacts]
[2023-12-31 22:41:52,478: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 23:22:58,125: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:22:58,127: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:22:58,132: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 23:22:58,134: INFO: common: created directory at: artifacts]
[2023-12-31 23:22:58,135: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 23:24:02,537: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:24:02,540: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:24:02,546: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 23:24:02,549: INFO: common: created directory at: artifacts]
[2023-12-31 23:24:02,550: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 23:25:22,716: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:25:22,720: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:25:22,732: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 23:25:22,732: INFO: common: created directory at: artifacts]
[2023-12-31 23:25:22,732: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 23:53:50,679: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:53:50,682: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:53:50,685: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 23:53:50,687: INFO: common: created directory at: artifacts]
[2023-12-31 23:53:50,688: INFO: common: created directory at: artifacts/data_validation]
[2023-12-31 23:54:10,071: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:54:10,073: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-31 23:54:10,079: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-12-31 23:54:10,081: INFO: common: created directory at: artifacts]
[2023-12-31 23:54:10,082: INFO: common: created directory at: artifacts/data_validation]
[2024-01-01 00:04:47,135: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 00:04:47,138: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 00:04:47,142: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 00:04:47,144: INFO: common: created directory at: artifacts]
[2024-01-01 00:04:47,146: INFO: common: created directory at: artifacts/data_validation]
[2024-01-01 00:19:17,901: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-01 00:19:17,901: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-01 00:19:18,834: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-01 00:19:18,834: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 00:19:18,834: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 00:19:18,834: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 00:19:18,834: INFO: common: created directory at: artifacts]
[2024-01-01 00:19:18,834: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-01 00:19:20,968: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 710E:36B2:B6C4FE:E54D25:6591B7AD
Accept-Ranges: bytes
Date: Sun, 31 Dec 2023 18:49:19 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10246-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1704048559.315369,VS0,VE317
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: fbc372783461699383e084e1842d936a7c91f482
Expires: Sun, 31 Dec 2023 18:54:19 GMT
Source-Age: 0

]
[2024-01-01 00:19:20,999: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-01 00:19:20,999: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-01 00:19:20,999: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 00:19:20,999: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 00:19:21,015: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 00:19:21,015: INFO: common: created directory at: artifacts]
[2024-01-01 00:19:21,015: INFO: common: created directory at: artifacts/data_validation]
[2024-01-01 00:19:21,093: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-01 02:45:37,239: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:45:37,242: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:45:37,247: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:45:37,250: INFO: common: created directory at: artifacts]
[2024-01-01 02:45:37,252: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:46:24,612: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:46:24,615: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:46:24,618: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:46:24,620: INFO: common: created directory at: artifacts]
[2024-01-01 02:46:24,622: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:50:14,395: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:50:14,399: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:50:14,402: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:50:14,404: INFO: common: created directory at: artifacts]
[2024-01-01 02:50:14,406: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:50:46,052: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:50:46,055: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:50:46,059: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:50:46,061: INFO: common: created directory at: artifacts]
[2024-01-01 02:50:46,063: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:50:46,173: INFO: 732983346: Splited data into train and test sets]
[2024-01-01 02:55:30,882: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:55:30,886: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:55:30,889: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:55:30,890: INFO: common: created directory at: artifacts]
[2024-01-01 02:55:30,891: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:55:30,978: INFO: 3647971484: Splited data into train and test sets]
[2024-01-01 02:55:37,310: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:55:37,313: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:55:37,317: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:55:37,318: INFO: common: created directory at: artifacts]
[2024-01-01 02:55:37,320: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:55:37,415: INFO: 3647971484: Splited data into train and test sets]
[2024-01-01 02:55:37,882: INFO: 3647971484: Balanced the training data]
[2024-01-01 02:57:02,623: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:57:02,626: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:57:02,630: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:57:02,631: INFO: common: created directory at: artifacts]
[2024-01-01 02:57:02,635: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 02:57:02,692: INFO: 3689143997: Splited data into train and test sets]
[2024-01-01 02:57:02,891: INFO: 3689143997: Balanced the training data]
[2024-01-01 02:59:30,731: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:59:30,735: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 02:59:30,739: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 02:59:30,740: INFO: common: created directory at: artifacts]
[2024-01-01 02:59:30,741: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:00:10,311: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:00:10,314: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:00:10,317: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:00:10,319: INFO: common: created directory at: artifacts]
[2024-01-01 03:00:10,320: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:02:34,007: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:02:34,010: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:02:34,012: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:02:34,014: INFO: common: created directory at: artifacts]
[2024-01-01 03:02:34,015: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:02:52,073: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:02:52,077: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:02:52,080: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:02:52,082: INFO: common: created directory at: artifacts]
[2024-01-01 03:02:52,083: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:02:52,302: INFO: 578451359: Data split and balanced successfully.]
[2024-01-01 03:05:17,208: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:05:17,211: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:05:17,215: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:05:17,216: INFO: common: created directory at: artifacts]
[2024-01-01 03:05:17,218: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:05:17,428: INFO: 578451359: Data split and balanced successfully.]
[2024-01-01 03:08:54,166: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:08:54,166: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:08:54,175: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:08:54,175: INFO: common: created directory at: artifacts]
[2024-01-01 03:08:54,175: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:08:54,392: INFO: 2594490729: Data split and balanced successfully.]
[2024-01-01 03:09:58,019: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:09:58,019: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:09:58,027: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:09:58,029: INFO: common: created directory at: artifacts]
[2024-01-01 03:09:58,031: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:09:58,226: INFO: 429273344: Data split and balanced successfully.]
[2024-01-01 03:42:30,263: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-01 03:42:30,263: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-01 03:45:29,974: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-01 03:45:29,974: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-01 03:45:31,302: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-01 03:45:31,302: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:45:31,302: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:45:31,302: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:45:31,302: INFO: common: created directory at: artifacts]
[2024-01-01 03:45:31,302: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-01 03:45:33,841: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: FD72:07D9:D0FA2B:1023FF3:6591E804
Accept-Ranges: bytes
Date: Sun, 31 Dec 2023 22:15:33 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10244-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1704060932.367250,VS0,VE1414
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: e233bcb8ca505f6d6bc646deedb65600bed5e60b
Expires: Sun, 31 Dec 2023 22:20:33 GMT
Source-Age: 0

]
[2024-01-01 03:45:33,857: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-01 03:45:33,857: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-01 03:45:33,857: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:45:33,857: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:45:33,857: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:45:33,857: INFO: common: created directory at: artifacts]
[2024-01-01 03:45:33,857: INFO: common: created directory at: artifacts/data_validation]
[2024-01-01 03:45:33,889: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-01 03:45:33,889: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-01 03:45:33,904: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:45:33,904: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:45:33,904: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:45:33,904: INFO: common: created directory at: artifacts]
[2024-01-01 03:45:33,904: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:45:33,904: ERROR: stage_03_data_transformation: An error occurred: 'DataTransformation' object has no attribute 'train_test_spliting']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_03_data_transformation.py", line 21, in main
    data_transformation.train_test_spliting()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataTransformation' object has no attribute 'train_test_spliting'
[2024-01-01 03:45:33,904: ERROR: main: An error occurred: 'DataTransformation' object has no attribute 'train_test_spliting']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 30, in <module>
    data_transformation.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_03_data_transformation.py", line 27, in main
    raise e
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_03_data_transformation.py", line 21, in main
    data_transformation.train_test_spliting()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataTransformation' object has no attribute 'train_test_spliting'
[2024-01-01 03:46:47,963: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-01 03:46:47,963: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-01 03:46:49,236: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-01 03:46:49,236: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:46:49,236: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:46:49,236: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:46:49,236: INFO: common: created directory at: artifacts]
[2024-01-01 03:46:49,236: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-01 03:46:50,009: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: FD72:07D9:D0FA2B:1023FF3:6591E804
Accept-Ranges: bytes
Date: Sun, 31 Dec 2023 22:16:49 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10240-MAA
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1704061010.956068,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: bfdeaa29832bb8dfb332fe3b462788357d0e8fd2
Expires: Sun, 31 Dec 2023 22:21:49 GMT
Source-Age: 77

]
[2024-01-01 03:46:50,021: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-01 03:46:50,021: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-01 03:46:50,021: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:46:50,021: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:46:50,021: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:46:50,021: INFO: common: created directory at: artifacts]
[2024-01-01 03:46:50,028: INFO: common: created directory at: artifacts/data_validation]
[2024-01-01 03:46:50,044: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-01 03:46:50,060: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-01 03:46:50,060: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:46:50,060: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 03:46:50,060: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 03:46:50,060: INFO: common: created directory at: artifacts]
[2024-01-01 03:46:50,060: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 03:46:50,496: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-01 03:46:50,496: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-01 19:32:11,273: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:32:11,297: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:32:11,320: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:32:11,324: INFO: common: created directory at: artifacts]
[2024-01-01 19:32:11,324: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:41:46,584: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:41:46,592: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:41:46,600: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:41:46,600: INFO: common: created directory at: artifacts]
[2024-01-01 19:41:46,600: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:43:50,181: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:43:50,184: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:43:50,187: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:43:50,189: INFO: common: created directory at: artifacts]
[2024-01-01 19:43:50,191: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:45:37,980: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:45:37,980: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:45:37,988: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:45:37,988: INFO: common: created directory at: artifacts]
[2024-01-01 19:45:37,988: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:48:02,170: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:48:02,170: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:48:02,178: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:48:02,178: INFO: common: created directory at: artifacts]
[2024-01-01 19:48:02,186: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:49:13,678: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:49:13,682: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:49:13,687: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:49:13,690: INFO: common: created directory at: artifacts]
[2024-01-01 19:49:13,691: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:50:37,734: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:50:37,738: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:50:37,741: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:50:37,742: INFO: common: created directory at: artifacts]
[2024-01-01 19:50:37,744: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:51:18,892: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:51:18,896: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:51:18,900: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:51:18,903: INFO: common: created directory at: artifacts]
[2024-01-01 19:51:18,904: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 19:52:16,906: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:52:16,914: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 19:52:16,914: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 19:52:16,914: INFO: common: created directory at: artifacts]
[2024-01-01 19:52:16,922: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 20:14:50,470: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-01 20:14:50,470: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-01 20:14:54,148: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-01 20:14:54,156: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:54,156: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:54,156: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 20:14:54,156: INFO: common: created directory at: artifacts]
[2024-01-01 20:14:54,156: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-01 20:14:56,189: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 18B2:3B55:11E87DB:16235E1:6592CFE7
Accept-Ranges: bytes
Date: Mon, 01 Jan 2024 14:44:56 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10229-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1704120296.798185,VS0,VE376
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 4b58fde4c86c1e6fb832729a0684a8a67af935aa
Expires: Mon, 01 Jan 2024 14:49:56 GMT
Source-Age: 0

]
[2024-01-01 20:14:56,222: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-01 20:14:56,222: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-01 20:14:56,230: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:56,230: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:56,238: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 20:14:56,238: INFO: common: created directory at: artifacts]
[2024-01-01 20:14:56,238: INFO: common: created directory at: artifacts/data_validation]
[2024-01-01 20:14:56,327: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-01 20:14:56,327: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-01 20:14:56,327: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:56,335: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:56,335: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 20:14:56,335: INFO: common: created directory at: artifacts]
[2024-01-01 20:14:56,335: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-01 20:14:56,967: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-01 20:14:56,967: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-01 20:14:56,967: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-01 20:14:56,983: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:56,983: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 20:14:56,991: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 20:14:56,991: INFO: common: created directory at: artifacts]
[2024-01-01 20:14:56,991: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-01 20:14:57,073: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-01 23:27:26,884: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:27:26,890: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:27:26,898: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 23:27:26,898: INFO: common: created directory at: artifacts]
[2024-01-01 23:27:26,898: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-01 23:28:44,252: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:28:44,254: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:28:44,263: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 23:28:44,263: INFO: common: created directory at: artifacts]
[2024-01-01 23:28:44,263: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-01 23:30:00,485: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:30:00,485: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:30:00,493: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 23:30:00,493: INFO: common: created directory at: artifacts]
[2024-01-01 23:30:00,493: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-01 23:31:48,027: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:31:48,033: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-01 23:31:48,035: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-01 23:31:48,035: INFO: common: created directory at: artifacts]
[2024-01-01 23:31:48,035: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:10:35,678: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:10:35,678: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:10:41,612: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:10:41,615: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:41,618: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:41,619: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:10:41,619: INFO: common: created directory at: artifacts]
[2024-01-02 00:10:41,621: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:10:43,115: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4312:63A2:125D520:16B11DB:6593072A
Accept-Ranges: bytes
Date: Mon, 01 Jan 2024 18:40:43 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10249-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1704134443.781222,VS0,VE329
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 1056ca8f4188f6ce3a4cf1ed00f3174e60565951
Expires: Mon, 01 Jan 2024 18:45:43 GMT
Source-Age: 0

]
[2024-01-02 00:10:43,133: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:10:43,133: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:10:43,136: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,140: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,141: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:10:43,143: INFO: common: created directory at: artifacts]
[2024-01-02 00:10:43,143: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:10:43,196: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:10:43,196: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:10:43,204: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,206: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,208: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:10:43,208: INFO: common: created directory at: artifacts]
[2024-01-02 00:10:43,210: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:10:43,708: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:10:43,709: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:10:43,709: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:10:43,713: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,716: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,718: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:10:43,719: INFO: common: created directory at: artifacts]
[2024-01-02 00:10:43,719: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:10:43,806: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:10:43,806: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:10:43,809: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,812: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:10:43,815: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:10:43,815: INFO: common: created directory at: artifacts]
[2024-01-02 00:10:43,816: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:10:45,021: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:12:10,858: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:12:10,859: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:12:14,123: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:12:14,126: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,128: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,130: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:12:14,131: INFO: common: created directory at: artifacts]
[2024-01-02 00:12:14,131: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:12:14,131: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 00:12:14,135: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:12:14,135: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:12:14,137: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,140: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,142: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:12:14,142: INFO: common: created directory at: artifacts]
[2024-01-02 00:12:14,142: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:12:14,177: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:12:14,177: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:12:14,181: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,183: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,186: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:12:14,187: INFO: common: created directory at: artifacts]
[2024-01-02 00:12:14,188: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:12:14,592: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:12:14,592: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:12:14,592: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:12:14,596: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,598: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,600: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:12:14,601: INFO: common: created directory at: artifacts]
[2024-01-02 00:12:14,601: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:12:14,654: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:12:14,654: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:12:14,657: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,661: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:12:14,664: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:12:14,665: INFO: common: created directory at: artifacts]
[2024-01-02 00:12:14,665: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:12:15,811: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:15:21,882: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:15:21,885: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:15:21,888: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:15:21,891: INFO: common: created directory at: artifacts]
[2024-01-02 00:15:21,893: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:17:46,208: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:17:46,213: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:17:46,217: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:17:46,219: INFO: common: created directory at: artifacts]
[2024-01-02 00:17:46,222: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:18:40,990: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:18:40,994: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:18:40,997: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:18:40,999: INFO: common: created directory at: artifacts]
[2024-01-02 00:18:41,000: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:18:42,026: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2024-01-02 00:23:30,259: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:23:30,259: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:23:33,252: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:23:33,252: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:33,252: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:33,252: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:23:33,252: INFO: common: created directory at: artifacts]
[2024-01-02 00:23:33,252: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:23:34,755: INFO: data_ingestion: artifacts/data_ingestion/data.zip download with following info: 
Connection: close
Content-Length: 68996
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "2e183ebd6a5b61282c4986b78d0fad9592ad3204a2291140e9b19adf0f43414c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4312:63A2:125D520:16B11DB:6593072A
Accept-Ranges: bytes
Date: Mon, 01 Jan 2024 18:53:34 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10251-MAA
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1704135214.447570,VS0,VE300
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 598e61e8f3826d22ebe2207b15c70c75899eefe1
Expires: Mon, 01 Jan 2024 18:58:34 GMT
Source-Age: 0

]
[2024-01-02 00:23:34,771: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:23:34,771: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:23:34,771: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:34,775: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:34,779: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:23:34,780: INFO: common: created directory at: artifacts]
[2024-01-02 00:23:34,780: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:23:34,813: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:23:34,813: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:23:34,821: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:34,821: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:34,821: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:23:34,829: INFO: common: created directory at: artifacts]
[2024-01-02 00:23:34,829: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:23:35,275: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:23:35,275: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:23:35,275: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:23:35,275: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:35,275: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:35,275: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:23:35,275: INFO: common: created directory at: artifacts]
[2024-01-02 00:23:35,275: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:23:35,355: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:23:35,355: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:23:35,355: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:35,363: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:23:35,363: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:23:35,363: INFO: common: created directory at: artifacts]
[2024-01-02 00:23:35,363: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:23:36,437: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:27:01,784: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:27:01,784: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:27:04,826: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:27:04,829: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:04,831: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:04,833: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:27:04,834: INFO: common: created directory at: artifacts]
[2024-01-02 00:27:04,834: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:27:04,834: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 00:27:04,838: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:27:04,838: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:27:04,841: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:04,845: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:04,847: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:27:04,848: INFO: common: created directory at: artifacts]
[2024-01-02 00:27:04,848: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:27:04,888: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:27:04,888: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:27:04,891: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:04,893: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:04,895: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:27:04,896: INFO: common: created directory at: artifacts]
[2024-01-02 00:27:04,896: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:27:05,287: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:27:05,288: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:27:05,289: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:27:05,290: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:05,293: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:05,295: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:27:05,295: INFO: common: created directory at: artifacts]
[2024-01-02 00:27:05,295: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:27:05,350: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:27:05,350: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:27:05,353: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:05,356: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:27:05,357: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:27:05,358: INFO: common: created directory at: artifacts]
[2024-01-02 00:27:05,358: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:27:06,416: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:31:01,529: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:31:01,529: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:31:04,829: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:31:04,829: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:04,829: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:04,829: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:31:04,829: INFO: common: created directory at: artifacts]
[2024-01-02 00:31:04,829: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:31:04,829: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 00:31:04,845: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:31:04,845: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:31:04,845: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:04,845: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:04,845: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:31:04,845: INFO: common: created directory at: artifacts]
[2024-01-02 00:31:04,845: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:31:04,877: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:31:04,877: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:31:04,892: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:04,892: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:04,892: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:31:04,892: INFO: common: created directory at: artifacts]
[2024-01-02 00:31:04,892: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:31:05,301: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:31:05,301: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:31:05,301: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:31:05,316: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:05,316: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:05,316: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:31:05,316: INFO: common: created directory at: artifacts]
[2024-01-02 00:31:05,316: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:31:05,379: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:31:05,379: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:31:05,380: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:05,380: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:31:05,380: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:31:05,380: INFO: common: created directory at: artifacts]
[2024-01-02 00:31:05,380: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:31:06,538: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:34:02,927: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:34:02,927: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:34:05,985: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:34:05,987: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:05,990: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:05,991: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:34:05,992: INFO: common: created directory at: artifacts]
[2024-01-02 00:34:05,992: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:34:05,992: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 00:34:05,995: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:34:05,995: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:34:05,998: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,000: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,002: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:34:06,003: INFO: common: created directory at: artifacts]
[2024-01-02 00:34:06,003: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:34:06,048: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:34:06,048: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:34:06,052: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,056: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,058: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:34:06,058: INFO: common: created directory at: artifacts]
[2024-01-02 00:34:06,059: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:34:06,465: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:34:06,466: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:34:06,466: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:34:06,466: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,466: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,466: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:34:06,466: INFO: common: created directory at: artifacts]
[2024-01-02 00:34:06,466: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:34:06,530: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:34:06,530: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:34:06,533: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,536: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:34:06,539: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:34:06,540: INFO: common: created directory at: artifacts]
[2024-01-02 00:34:06,540: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:34:07,712: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:35:49,228: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:35:49,228: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:35:52,449: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:35:52,451: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,455: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:35:52,456: INFO: common: created directory at: artifacts]
[2024-01-02 00:35:52,456: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:35:52,456: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 00:35:52,460: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:35:52,460: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:35:52,463: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,465: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,467: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:35:52,468: INFO: common: created directory at: artifacts]
[2024-01-02 00:35:52,468: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:35:52,492: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:35:52,492: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:35:52,492: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,511: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,513: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:35:52,513: INFO: common: created directory at: artifacts]
[2024-01-02 00:35:52,514: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:35:52,912: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:35:52,913: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:35:52,913: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:35:52,916: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,918: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,920: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:35:52,920: INFO: common: created directory at: artifacts]
[2024-01-02 00:35:52,921: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:35:52,978: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:35:52,978: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:35:52,981: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,984: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:35:52,986: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:35:52,987: INFO: common: created directory at: artifacts]
[2024-01-02 00:35:52,987: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:35:54,214: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 00:37:47,700: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 00:37:47,700: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 00:37:50,733: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 00:37:50,733: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:50,739: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:50,739: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:37:50,739: INFO: common: created directory at: artifacts]
[2024-01-02 00:37:50,739: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 00:37:50,739: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 00:37:50,739: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 00:37:50,739: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 00:37:50,739: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:50,755: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:50,755: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:37:50,755: INFO: common: created directory at: artifacts]
[2024-01-02 00:37:50,755: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 00:37:50,786: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 00:37:50,786: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 00:37:50,786: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:50,786: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:50,802: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:37:50,802: INFO: common: created directory at: artifacts]
[2024-01-02 00:37:50,802: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 00:37:51,194: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 00:37:51,194: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 00:37:51,194: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 00:37:51,194: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:51,194: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:51,194: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:37:51,210: INFO: common: created directory at: artifacts]
[2024-01-02 00:37:51,210: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 00:37:51,256: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 00:37:51,256: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 00:37:51,256: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:51,256: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 00:37:51,256: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 00:37:51,256: INFO: common: created directory at: artifacts]
[2024-01-02 00:37:51,256: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 00:37:52,337: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "D:\vehicle_insurance_fraud_detection\main.py", line 51, in <module>
    obj.main()
  File "D:\vehicle_insurance_fraud_detection\src\vifd\pipeline\stage_05_model_evaluation.py", line 14, in main
    model_evaluation_config.log_into_mlflow()
  File "d:\vehicle_insurance_fraud_detection\src\vifd\components\model_evaluation.py", line 30, in log_into_mlflow
    with mlflow.start_run():
         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
                     ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lahari\anaconda3\Lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2024-01-02 01:01:12,962: INFO: utils: Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 01:01:12,962: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 01:01:16,634: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 01:01:16,642: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:16,642: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:16,651: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 01:01:16,651: INFO: common: created directory at: artifacts]
[2024-01-02 01:01:16,651: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 01:01:16,651: INFO: data_ingestion: File already exists of size: ~ 67 KB]
[2024-01-02 01:01:16,659: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 01:01:16,659: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 01:01:16,667: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:16,667: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:16,667: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 01:01:16,667: INFO: common: created directory at: artifacts]
[2024-01-02 01:01:16,667: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 01:01:16,700: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 01:01:16,700: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 01:01:16,700: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:16,715: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:16,715: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 01:01:16,715: INFO: common: created directory at: artifacts]
[2024-01-02 01:01:16,715: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 01:01:17,090: INFO: data_transformation: Data split and balanced successfully.]
[2024-01-02 01:01:17,090: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 01:01:17,090: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2024-01-02 01:01:17,090: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:17,106: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:17,106: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 01:01:17,106: INFO: common: created directory at: artifacts]
[2024-01-02 01:01:17,106: INFO: common: created directory at: artifacts/model_trainer]
[2024-01-02 01:01:17,168: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 01:01:17,168: INFO: main: >>>>>>> stage Model Evaluation stage started <<<<<<<<]
[2024-01-02 01:01:17,168: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:17,168: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 01:01:17,168: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-01-02 01:01:17,168: INFO: common: created directory at: artifacts]
[2024-01-02 01:01:17,168: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 01:01:18,466: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2024-01-02 01:01:28,045: INFO: main: >>>>>>>>> stage Model Evaluation stage completed <<<<<<<<<\n=============x]
